var/spack/repos/builtin/packages/py-xyzservices/package.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class PyXyzservices(PythonPackage):
+    """xyzservices is a lightweight library providing a repository of
+    available XYZ services offering raster basemap tiles."""
+
+    homepage = "https://xyzservices.readthedocs.io/"
```

Comment:
```suggestion
    homepage = "https://github.com/geopandas/xyzservices"
```
We try to be consistent with what is specified on PyPI.